examples/plot_choose-thumbnail.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
If you could name this file `plot_choose_thumbnail.py` with underscores, it would be more consistent.